docs/Local-Dispatcher-Windows.md

```diff
@@ -1 +1,126 @@
+with additional notes, to help avoid problems.
+
+## Host name "aem-publish.local"
+
+Following the AEM instructions will set up your dispatcher on host "localhost" by default.  To
+follow the dispatcher experiments more closely, the host name "**aem-publish.local**" should be used.
+To do that, follow these instructions which loosely follow some of the steps found in this web page:
+[tutorialsteacher](https://www.tutorialsteacher.com/articles/set-custom-domain-name-for-localhost-website-iis)
+
+- In IIS Manager, right click on the project name (top level on the left-hand tree)
+- Click "Add Website..."
+- Enter:
+  - Site Name: **aem-publish.local**
+  - Physical Path: a path anywhere on your filesystem
+    - maybe create this folder close to your author & publish instances for convenience, or under \inetpub
+    - this is where the cached files will reside
+  - Bind to port: 8080
+  - Host Name: **aem-publish.local**
+- Edit your _hosts_ file
+  - This file is often found at "C:\Windows\System32\drivers\etc\"
+  - Add `127.0.0.1 aem-publish.local` at the end of the file
+- Try "http://aem-publish.local:8080" - this will produce poor results, but should produce a default server
+error response, which proves the site is up and running.
+- To avoid confusion, stop the "Default Web Site" site
+
+Continue with the instructions below, but instead of making changes to the default site, make the changes to the
+new **aem-publish.local** site.  That is your "Dispatcher" or "Dispatcher cache" site now.
+
+## Dispatcher Installation Instructions
+
+Follow the Windows/IIS instructions from the following documentation: 
+[Experience Manager Dispatcher Help - Installing Dispatcher](https://docs.adobe.com/content/help/en/experience-manager-dispatcher/using/getting-started/dispatcher-install.html)
+
+That documentation is mostly complete.  If any steps are confusing, do not skip them because the order
+is important.  Keep this README open beside that documentation, and work through them both together.
+
+NOTE: For IIS/Windows 10, "*Server Manager*" refers to the "Windows Features"
+
+### Microsoft Internet Information Server
+Be sure to install **ISAPI Extenstions** right away:
+  - Go to the `Turn Windows features on or off` panel
+  - Install `World Wide Web Services/Application and Development Features/ASAPI Extensions`
+    - If you don't, you will need to uninstall and start over again
+  - Install `World Wide Web Services/Application and Development Features/ASP.NET 4.8` (or most recent)
+  - Add other options as you see fit - do not remove any
+  - Install `World Wide Web Services/Security/Windows Authentication` feature
+  - A reboot may be necessary
+
+### Microsoft IIS - Configure the Dispatcher INI File
+
+```
+OPTIONAL disp_iis INI FILE HINT
+Set the log files to rotate to avoid large logs including old info:
+- logfile=C:\inetpub\logs\dispatcher\dispatcher_iis.%Y-%m-%d-%H_%M_%S.log
+- rotate=5M
```

Comment:
Does this rotate to a new log file every 5 minutes? That could be frustrating while debugging, if tailing the logs.